exampleCourse/courseInstances/SectionA/assessments/groupWork/template/infoAssessment.json

```diff
@@ -40,17 +40,17 @@
             "credit": 100
         }
     ],
+    "canView": ["Manager", "Reflector", "Recorder", "Contributor"],
     "zones": [
         {
             "title": "Fundamental questions",
             "bestQuestions": 3,
+            "canSubmit": ["Recorder"],
             "questions": [
                 {
                     "id": "demo/demoNewton-page1",
                     "points": 1, 
-                    "maxPoints": 5, 
-                    "canView": ["Manager", "Reflector", "Recorder", "Contributor"],
-                    "canSubmit": ["Recorder"]
```

Comment:
Nit: for this specific example, I think it's actually harder to parse who can do what, as you have to look at three separate levels in the hierarchy. I acknowledge that the functionality introduced by this PR is nice for power users or those with lots of questions that should be viewable by everyone, but I'd rather leave the existing permissions in place for this example.

Reply:
That makes sense! This feature definitely makes more sense for much larger assessments. I can revert the change back to how the permissions were originally laid out. Explanations can be given in documentation anyway!